experiments.tex

```diff
@@ -141,32 +142,32 @@ \subsubsection{Effect of Increased Compute workload for RMSD Task}
 \label{fig:tcomp_tIO_effect}
 \end{figure}
 
-Even for moderately compute-bound workloads such as the $40\times$ and $70\times$ RMSD tasks, increasing the computational workload over I/O reduced the impact of stragglers even though they still contributed to large variations in timing across different ranks and thus irregular scaling due to the fluctuations.
+Even for moderately compute-bound workloads, such as the $40\times$ and $70\times$ RMSD tasks, increasing the computational workload over I/O reduced the impact of stragglers. Although, they still contributed to large variations in timing across different ranks and thus irregular scaling.
```

Comment:
Don't start a sentence with "Although, ...": it should start a subordinate clause "Although they still ... , MAIN CLAUSE".

Bottomline: just leave the previous sentence, I think it's ok.

Reply:
fixed